go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/trustwallet/blockatlas
 go 1.12.6
 
 require (
+	github.com/davecgh/go-spew v1.1.1
```

Comment:
an extra?

Reply:
It added itself when I called the assert.Equal dep, thought it was an indirect dependency.

I'm removing it.